tests/api_fastapi/execution_api/routes/test_task_instances.py

```diff
@@ -716,6 +716,81 @@ def test_ti_runtime_checks_failure(self, client, session, create_task_instance):
 
         session.expire_all()
 
+    def test_ti_update_state_to_fail_with_asset_profile_extra(self, client, session, create_task_instance):
```

Comment:
You can just remove those tests.

The base class `StrictBaseModel` is tested. We would only need to check that the actual datamodel extends the `StrictBaseModel`. I don't think we should bother with that.